### ISSUE TITLE
Revert jump to the most recent revision

### ISSUE THREAD

**Issue by bradynpoulsen** [mark-hahn/live-archive#15]
I made a test file I'm tracking with live-archive. I have made several changes and saved after each one and I can view each of those changes as a different revision. When I find the revision I want (ie, Version 2 out of 5), I click "Revert" expecting it to update my test file to that point in history, but instead just jumps to Version 5 out of 5 and leaves the test file untouched.

Am I just expecting the wrong result of that button?

**Comment by mark-hahn** [comment]
No, it is supposed to work as you want.  It seems like live-archive is
being restarted.

What kind of file is test?  Can you send a copy to mark@hahnca.com?

**Comment by bradynpoulsen** [comment]
```php
<?php

class FooBar
{
    protected $name;

    public function __construct($name)
    {
        assert(func_num_args() === 1);
        assert('is_string($name)');

        $this->name = $name;
    }

    public function sayHello()
    {
        assert(func_num_args() === 0);

        echo "Hello there, {$this->name}";
    }
}
```
